test/tx_loc_modes2.ml

```diff
@@ -0,0 +1,32 @@
+(* Location is in Lock-free mode
+   Each Thread increments the value at a and reads a in a loop*)
+
+module Loc = Kcas.Loc
+module Mode = Kcas.Mode
+module Tx = Kcas.Tx
+
+let a = Loc.make ~mode:Mode.lock_free 4
```

Comment:
You could combine the two programs into one by reading the mode from the command line arguments.

It would also be useful to read other parameters from the command line.

Something simple like this should do:

```ocaml
let loop_count = try int_of_string Sys.argv.(1) with _ -> 750
let mode = try if Sys.argv.(2) = "lock-free" then Some Mode.lock_free else None with _ -> None
(* ... *)
let a = Loc.make ?mode 4
```